src/db/tables/onerep_scans.ts

```diff
@@ -20,12 +20,16 @@ async function getOnerepScanResults(
   >;
 }
 
-async function getLatestOnerepScan(onerepProfileId: number): Promise<{
+export interface GetLatestOnerepScanResult {
```

Comment:
nit: I think this is the only interface prefixed w/ `Get`, per:

<details>
<summary><kbd>git grep -Pho "\binterface .*? {" | sort | uniq</kbd></summary>

```sh
interface Breach {
interface BreachResolution {
interface BreachResolutionRequest {
interface BreachRow {
interface BreachStats {
interface ComboBoxProps extends ComboBoxStateOptions<object> {
interface DataFileUrls {
interface EmailAddRequest {
interface EmailAddressRow {
interface EmailDeleteRequest {
interface EmailResendRequest {
interface EmailUpdateCommOptionRequest {
interface FeatureFlagRow {
interface FxaProfile {
interface GetLatestOnerepScanResult {
interface IntrinsicElements {
interface ItemObject {
interface JWT {
interface JwtPayload {
interface ListBoxProps extends AriaListBoxOptions<unknown> {
interface OnerepScanRow {
interface OnerepWebhookRequest {
interface OptionProps extends AriaListBoxOptions<unknown> {
interface PasswordChangeEvent {
interface PopoverProps extends AriaPopoverProps {
interface Profile {
interface ProfileChangeEvent {
interface ProgressBarProps extends AriaProgressBarProps {
interface Props extends ComponentProps<"button"> {
interface RelevantLocation {
interface RelevantLocationAlternate {
interface RequestBreachScanErrorResponse {
interface RequestBreachScanSuccessResponse {
interface ScanProgressBody {
interface SearchLocationParams {
interface SearchLocationResults {
interface Session {
interface Subscriber {
interface SubscriberBreachResolution {
interface SubscriberEmail {
interface SubscriberRow {
interface SubscriptionStateChangeEvent {
interface Tables {
interface UserInfo {
interface VerifiedEmail {
interface WelcomeScanBody {
interface Window {
```
</details>